app/src/hooks/useGrowthbookIntegration.ts

```diff
@@ -13,7 +13,7 @@ const useGrowthBookIntegration = () => {
   const userAttributes = useSelector(getUserAttributes);
 
   const usePrevious = (value: any) => {
-    const ref = useRef();
+    const ref = useRef<unknown>(undefined);
```

Comment:
Is this really required?

Reply:
Needed initial value, type can be ignore, updated